src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -190,9 +197,14 @@ export default {
       return nextState;
     },
   },
+  vaFacilityV2: {
+    url: '/new-appointment/va-facility-2',
+  },
   clinicChoice: {
     url: '/new-appointment/clinics',
-    previous: 'vaFacility',
+    previous(state) {
+      return getFacilityPageKey(state);
+    },
```

Comment:
Checking the feature flag requires access to our redux state, so in this case we convert this `previous` key to return a function instead of just the `vaFacility` string.